### PR TITLE
[foundryvtt] Fix typo on spelling of foundry

### PR DESCRIPTION
**Pull request by Truxnell**
This PR includes a 'simple' change to fix some typos in the foundryvtt chart.  The folder and a few paths/docs were spelt 'founderyvtt' not 'foundvtt'.  This broke the hyperlink from the chart base readme.md and was a little confusing at places.

Aligns folder and naming to projects correct name, incidentally fixing broken links.

If I've missed something, might break a users existing config.  That said, to me it looks like the important aspects of the chart were spelt correctly, so I Dont envisage the change to be a breaking change.

I don't believe there is limitations.  I have not bumped the chart version in this fix as its should be quite transparent to end users as its a more internal adjustment.

- fixes #

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [ ] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Comment by onedr0p** [comment]
@Truxnell thanks for the update, but it will also need an update to this file too:

https://github.com/k8s-at-home/charts/blob/master/.github/ct-install.yaml#L16

**Comment by Truxnell** [comment]
Thanks for the catch @onedr0p!.   I had fixed that (and a typo in the README.md and a comment in values.yaml) but I seem to have missed pushing them in the original commit.

I have pushed another commit to fix these 3 files.

File: .github/ct-install.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ excluded-charts:
 - charts/stable/ser2sock
 - charts/stable/zalando-postgres-cluster
 - charts/stable/zigbee2mqtt
-- charts/stable/founderyvtt
+- charts/stable/foundeyvtt
```
Comment (bjw-s) [inline .github/ct-install.yaml:16]:
```suggestion
- charts/stable/foundryvtt
```

**Comment by bjw-s** [comment]
Uh-Oh... I think I fudged your branch history... I'll try to fix that!

**Review by bjw-s — APPROVED** [review]

**Comment by bjw-s** [comment]
@all-contributors please add @Truxnell for code

**Comment by Truxnell** [comment]
Appreciated @bjw-s but not necessary for such a small PR!

**Comment by bjw-s** [comment]
Every contribution is a welcomed one :)

**Pull request merged**